### ISSUE TITLE
Todos before stable release of v3

### ISSUE THREAD

**Issue by dkundel** [twilio-labs/serverless-toolkit#255]
These are some of the scenarios we should double check before we release the stable release:

- [x] Create a project from scratch using the `plugin-serverless@next`
- [x] Deploy new project using `plugin-serverless@next` from default CLI profile
- [x] Deploy new project using `plugin-serverless@next` with manual profile using `-p`
- [x] Deploy new project using `plugin-serverless@next` with --username and --password
- [x] Create a project from scratch with `create-twilio-function@next`
- [x] Deploy an old project using `plugin-serverless@next` from default CLI profile
- [x] Deploy an old project using `plugin-serverless@next` with manual profile using `-p`
- [x] Deploy an old project using `plugin-serverless@next` with --username and --password
- [x] Try migration script on existing project
- [x] Configure different `env` files for different environments in `.twilioserverlessrc` and deploy each.
- [x] Configure different `serviceSid` for different projects in `.twilioserverlessrc` and deploy each.
- [x] Configure two different `env` files for `start` and `deploy` commands in `.twilioserverlessrc` and verify they get picked up
- [x] Update the docs at twilio.com/docs/labs/serverless-toolkit to remove notion of `.twilio-functions` and update the way credentials work.

**Comment by philnash** [comment]
Well, bad start. I started on the first point, loaded the example functions from `create-twilio-function` and the private example wasn't working. Turned out the forking code didn't keep track of the functions and assets across the process boundary, so `Runtime.getAssets()` and `Runtime.getFunctions()` were just returning `{}`.  Opened #256 to fix it.

Otherwise, creating a new project works! Checking that off.
